app_dart/lib/src/service/github_service.dart

```diff
@@ -7,35 +7,55 @@ import 'dart:convert' show json;
 import 'package:github/github.dart';
 import 'package:http/http.dart';
 
-import '../datastore/cocoon_config.dart';
-
 class GithubService {
   const GithubService(this.github);
 
   final GitHub github;
 
-  /// Lists the commits of the provided repository [slug] and [branch].
+  /// Lists commits of the provided repository [slug] and [branch]. When
+  /// [lastCommitTimestampMills] equals 0, it means a new release branch is
+  /// found and only the branched commit will be returned. Otherwise, it returns
+  /// all newer commits since [lastCommitTimestampMills].
   Future<List<RepositoryCommit>> listCommits(
```

Comment:
The description doesn't mention the fact that when a new release branch is found, there can be more than one branched commits in rare cases.

Reply:
Updated.